app/plugins/eu.esdihumboldt.hale.app.transform/src/eu/esdihumboldt/hale/app/transform/ExecApplication.groovy

```diff
@@ -134,7 +135,8 @@ $baseCommand
          [-providerId <ID-of-target-writer>]
          [<setting>...]
 	 [-filter <filter-expression>]
-	 [-filter-on <Type>:<filter-expression>]
+	 [-filter-on <Type> <filter-expression>]
+ 	 [-args-file <Path>]
```

Comment:
Maybe put that at the very beginning (even before `-project`), as this may replace the need to provide any of the other arguments (if they are provided in the file).